engine/engine-paper/src/main/kotlin/com/typewritermc/engine/paper/interaction/ChatHistoryHandler.kt

```diff
@@ -203,19 +247,24 @@ class ChatHistory {
                         msg = msg.merge(message)
                         continue
                     }
+                    ResendTokenRegistry.issue(player, msg.message)
```

Comment:
This is a lot of duplication. Why not add the issue thing in the send function of the message?